Revise Fusion content (#137)
seqeralabs/docs: fusion_docs/faq.mdx

# Frequently Asked Questions

**Cloud object storage providers supported by Fusion**

Fusion currently supports AWS S3, Azure Blob Storage, and Google Cloud Storage.

**Fusion support for MinIO**

[MinIO](https://min.io/), which implements an S3-compatible API, is supported in place of AWS S3.

> See the documentation how to configure your pipeline execution to use Fusion and Minio. (link to guide TBD).

Reviewer: ```suggestion
See the documentation how to configure your pipeline execution to use Fusion and MinIO. (link to guide TBD).
```

> **Fusion as a separate download**

Reviewer 2: By definition a FAQ is organised in question

Currently, Fusion can be used only by enabling Wave containers in the configuration of your Nextflow pipeline. Fusion is not available as a separate download.

**Wave containers as a prerequisite for using Fusion**

> Fusion works at the level of Netflow job execution. For this reason, it needs to run in containerized job execution context.

Reviewer: ```suggestion
Fusion works at the level of Nextflow job execution. For this reason, it needs to run in a containerized job execution context.
```

> This would require to rebuild all containers used by your data pipeline to include the Fusion client each time a new version of the Fusion client is released, and it would make necessary to maintain a custom mirror or existing containers collections, such as [BioContainers](https://biocontainers.pro/) which is definitively not desirable.

Reviewer: ```suggestion
When a new Fusion client version is released, you'll need to rebuild all the containers used by your data pipeline to include the new version. This makes it necessary to maintain a custom mirror or existing container collections, such as [BioContainers](https://biocontainers.pro/) which is not recommended.
```

> Wave allows adding the Fusion client in your pipeline containers at deploy time, without having to rebuild them or to maintainer a separate container image collection.

Reviewer: ```suggestion
Wave allows adding the Fusion client in your pipeline containers at deploy time, without having to rebuild them or to maintaine a separate container image collection.
```
